webservices/common/models/costs.py

```diff
@@ -6,34 +6,40 @@
 class CommunicationCost(db.Model):
     __tablename__ = 'ofec_communication_cost_mv'
 
-    idx = db.Column(db.Integer, primary_key=True)
-    committee_id = db.Column('cmte_id', db.String, index=True)
+    sub_id = db.Column(db.Integer, primary_key=True)
     candidate_id = db.Column('cand_id', db.String, index=True)
-    committee_name = db.Column('cmte_nm', db.String)
-    candidate_name = db.Column('cand_name', db.String)
-    candidate_office_state = db.Column('cand_office_st', db.String, index=True)
-    state_full = db.Column('st_desc', db.String)
-    candidate_office_district = db.Column('cand_office_district', db.String, index=True)
-    candidate_office = db.Column('cand_office', db.String, index=True)
-    candidate_party_affiliation = db.Column('cand_pty_affiliation', db.String, index=True)
-    party_full = db.Column('pty_desc', db.String)
-    transaction_date = db.Column('transaction_dt', db.Date, index=True)
-    transaction_amount = db.Column('transaction_amt', db.Numeric(30, 2), index=True)
+    committee_id = db.Column('cmte_id', db.String, index=True)
+    pdf_url = db.Column(db.String)
+    candidate_name = db.Column('s_o_cand_nm', db.String)
+    candidate_last_name = db.Column('s_o_cand_l_nm', db.String)
+    candidate_middle_name = db.Column('s_o_cand_m_nm', db.String)
+    candidate_first_name = db.Column('s_o_cand_f_nm', db.String)
+    candidate_office_state = db.Column('s_o_cand_office_st', db.String, index=True)
+    state_full = db.Column('s_o_cand_office_st_desc', db.String)
+    candidate_office_district = db.Column('s_o_cand_office_district', db.String, index=True)
+    candidate_office = db.Column('s_o_cand_office', db.String, index=True)
+    candidate_office_full =db.Column('s_o_cand_office_desc', db.String)
+    transaction_date = db.Column('communication_dt', db.Date, index=True)
+    transaction_amount = db.Column('communication_cost', db.Numeric(30, 2), index=True)
     transaction_type = db.Column('transaction_tp', db.String)
-    receipt_date = db.Column('f7_receipt_dt', db.String)
-    purpose = db.Column(db.String)
     communication_type = db.Column('communication_tp', db.String, index=True)
+    communication_type_full = db.Column('communication_tp_desc', db.String)
```

Comment:
Thanks for the verification @PaulClark2, I believe the schemas should be quite similar so it's no big deal.